cmd/genyaml/gen_kubectl_yaml.go

```diff
@@ -64,8 +65,7 @@ func main() {
 	// Set environment variables used by kubectl so the output is consistent,
 	// regardless of where we run.
 	os.Setenv("HOME", "/home/username")
```

Comment:
just curious , can we make /home/username as variable somewhere ?

Reply:
That is definitely an unexpected line of code. It might be good to dig through the history and find someone who can explain it more.